src/checkbox/checkbox-group.svelte

```diff
@@ -0,0 +1,64 @@
+<script>
+  import classes from '../_utils/classes.js';
+  import getColorPickerStyles from '../_utils/color-picker-styles.js';
+  import s from '../_utils/plural-s.js';
+  import Checkbox from './checkbox.svelte';
+
+  let _class = null;
+  export { _class as class };
+  export let checkboxClass = null;
+  export let labelClass = null;
+
+  export let color = false;
+
+  export let items;
+  export let name;
+  export let labelsLeft = false;
+  export let max = null;
+
+  $: currentChecked = items.reduce((acc, elt) => acc + elt.checked, 0);
+  const maxReachedTooltip = { title: `Can only select ${max} value${s(max)}.` };
```

Comment:
Perhaps this should be a prop to allow for better customization and i18n.

Reply:
Will add as a future improvement